module3/homework/vectorOfSharedPointers/vectorFunctions.cpp

```diff
@@ -0,0 +1,32 @@
+#include "vectorFunctions.hpp"
+#include <iostream>
+
+std::vector<std::shared_ptr<int>> generate(int count){
+    std::vector<std::shared_ptr<int>> newVector;
+    newVector.reserve(count);
+    for (size_t i = 0 ; i < count ; ++i) {
+        newVector.emplace_back(std::make_shared<int>(i));
+    }
+    return newVector;
+}
+void print(std::vector<std::shared_ptr<int>>& newVector){
```

Comment:
I would add one empty line between `}` and `void print(std::vector<std::shared_ptr<int>>& newVector){` but it's just my personal opinion. ;) The same for other functions.